src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -180,6 +193,28 @@ public StudentAttributes updateStudentCascade(StudentAttributes.UpdateOptions up
         return updatedStudent;
     }
 
+    public List<StudentAttributes> updateStudentCascadeBatch(List<StudentAttributes.UpdateOptions> updateOptionsList)
+            throws CascadingTransactionException {
+        StudentsDb.BatchUpdateStudentsTransaction studentUpdateTransaction =
+                new StudentsDb.BatchUpdateStudentsTransaction(studentsDb, updateOptionsList);
+        List<StudentUpdate> studentUpdates = studentUpdateTransaction.getUpdatedStudents();
```

Comment:
This is always going to be empty so there is no updates even happening below for the other functions.